saleor/checkout/migrations/0040_add_handle_checkouts_permission.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 3.2.12 on 2022-03-08 10:35
+
+from django.core.management.sql import emit_post_migrate_signal
+from django.db import migrations
+
+
+def assing_permissions(apps, schema_editor):
+    # force post signal as permissions are created in post migrate signals
+    # related Django issue https://code.djangoproject.com/ticket/23422
+    emit_post_migrate_signal(2, False, "default")
+    Permission = apps.get_model("auth", "Permission")
+    App = apps.get_model("app", "App")
+
+    handle_checkout = Permission.objects.filter(
+        codename="handle_checkouts", content_type__app_label="checkout"
+    ).first()
+    apps = App.objects.filter(
+        permissions__content_type__app_label="checkout",
+        permissions__codename="manage_checkouts",
+    )
+
+    for app in apps.iterator():
+        app.permissions.add(handle_checkout)
+
```

Comment:
Groups with `manage_checkouts` should also get this permission.
Here is a related commit: https://github.com/saleor/saleor/pull/9393/commits/395b52a7364b402c228bc4f5e4e10af1815897b3

Reply:
Thanks Iga! <3 My bad